dist/stat_data.py

```diff
@@ -1087,6 +1087,7 @@ def __init__(self, name, desc, flags=''):
     CheckpointStat('checkpoint_cleanup_pages_evict', 'pages added for eviction during checkpoint cleanup'),
     CheckpointStat('checkpoint_cleanup_pages_obsolete_tw', 'pages dirtied due to obsolete time window by checkpoint cleanup'),
     CheckpointStat('checkpoint_cleanup_pages_removed', 'pages removed during checkpoint cleanup'),
+    CheckpointStat('checkpoint_cleanup_pages_selected_reclaim_space', 'pages selected during checkpoint cleanup (reclaim_space)'),
```

Comment:
We will read the page into the cache without skipping it. How about mentioning `read` in the stat name? 
`pages read into the cache ...`?

Reply:
Sure, see 5a889d064b2828d725ed4ac8f9d3aa0b11f3e972